### PR TITLE
EOS-23339: Integration of py-utils unittests into py-utils test_framework.

### PR DESCRIPTION
**Pull request by veerendra-simha-garikipati**
Signed-off-by: Veerendra Garikipati <veerendra.garikipati@seagate.com>

# Problem Statement
- Enhancements and Integration of py-utils unittests into py-utils test_framework.

# Design
-  https://jts.seagate.com/browse/EOS-23339

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: Y
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]: NA
- [x] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/test/cli_framework/test_cli_framework.py [hunk 1]
```diff
@@ -25,30 +25,32 @@
...
             'users': { 'list': True, 'update': True, 'create': True }
             }
-        self.directory_path = os.path.join("test_data/")
-        self.cmd_response = payload.CommonPayload(os.path.join("test_data/","users_command_output.json")).load()
-        self.expected_output_file = os.path.join("test_data/","users_show_table_output")
-        self.obtained_output_file = os.path.join("/tmp","users_show_table_obtained_output")
+        cls.directory_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test_data')
+        cls.cmd_response = payload.CommonPayload(os.path.join(cls.directory_path,'users_command_output.json')).load()
+        cls.expected_output_file = os.path.join(cls.directory_path,'users_show_table_output')
+        cls.obtained_output_file = os.path.join('/tmp','users_show_table_obtained_output')
```
Comment (vastradparayya) [inline py-utils/test/cli_framework/test_cli_framework.py:38]: spaces after comma

File: py-utils/test/elastic_search/test_elasticsearch.py [hunk 2]
```diff
@@ -22,10 +22,12 @@
...
-        self.elasticsearch = Elasticsearch(host='localhost', port=9200)
-        self.index = "elasticsearch_test"
-        self.data = {
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.elasticsearch = Elasticsearch(host='localhost', port=9200)
+        cls.index = "elasticsearch_test"
+        cls.data = {
             "test": "elasticsearch_service"
```
Comment (vastradparayya) [inline py-utils/test/elastic_search/test_elasticsearch.py:31]: Single quotes

File: py-utils/test/validator/test_path_validator.py [hunk 3]
```diff
@@ -32,8 +32,10 @@ def test_path(self):
 
 	def test_remote_path(self):
 		"""Check paths."""
-
-		PathV().validate('exists', self.paths, self.host)
+		try:
+			PathV().validate('exists', self.paths, self.host)
+		except Exception:
```
Comment (vastradparayya) [inline py-utils/test/validator/test_path_validator.py:37]: Indentation error

File: py-utils/test/elastic_search/test_elasticsearch.py (hunk 2 above)
Comment (veerendra-simha-garikipati) [reply py-utils/test/elastic_search/test_elasticsearch.py]: Done

File: py-utils/test/validator/test_path_validator.py (hunk 3 above)
Comment (veerendra-simha-garikipati) [reply py-utils/test/validator/test_path_validator.py]: Done

File: py-utils/test/setup.py [hunk 4]
```diff
@@ -47,24 +47,25 @@
...
         "": [
             "*.json",
             "plans/*.pln",
             "schema/*.json",
             "kv_store/*.json",
-            "conf_store/*.json",
+            "conf_store/*",
             "message_bus/*.conf",
-            "cli_framework/test_data/*.json",
+            "cli_framework/test_data/*",
```
[the remark this answers is not in the document]
Comment (veerendra-simha-garikipati) [reply py-utils/test/setup.py:69]: clis_framework directory contains .json files and a file without any extension, so updated cli_framework/test_data/* to include all the files.

**Pull request merged**